FrameworkSystem/Client/ComponentInstaller.py

```diff
@@ -730,6 +736,8 @@ def addDefaultOptionsToCS(self, gConfig_o, componentType, systemName,
       for element in execList:
         result = self.addDefaultOptionsToCS(gConfig_o, componentType, systemName, element, extensions, self.setup,
                                             {}, overwrite)
+        if not result['OK']:
```

Comment:
Is that not a reason to exit ?

Reply:
I don't know, I didn't want to change the logic here, not for this PR at least.